go.mod

```diff
@@ -1,9 +1,10 @@
-module github.com/eliukblau/pixterm/pkg/ansimage
+module github.com/eliukblau/pixterm
 
-go 1.13
+go 1.12
```

Comment:
KEEP THIS WITHOUT CHANGES.

In Go 1.13, modules are "officially" supported. But in 1.12 are "experimentally" supported.

Reply:
I restored this line by myself.